mono/utils/mono-threads-mach-helper.c

```diff
@@ -24,7 +24,8 @@
 */
 void mono_threads_init_dead_letter (void);
 void mono_threads_install_dead_letter (void);
-void mono_thread_info_detach (void);
+MONO_API void
```

Comment:
This may be too much unwarranted change, vs. just G_BEGIN_DECLS / G_END_DECLS.

Reply:
Well..implementation is elsewhere, so I guess this isn't really a change in import/export.